packages/blockchain-wallet-v4-frontend/src/modals/SfoxExchangeData/Create/AcceptTerms.js

```diff
@@ -105,44 +105,60 @@ class AcceptTerms extends Component {
             <FieldsContainer>
               <FieldContainer>
                 <Text size='14px' style={spacing('mb-10')}>
-                  <FormattedMessage id='sfoxexchangedata.create.createaccount.partner.verifiedemail' defaultMessage='Verified Email Address' />
+                  {
+                    emailVerified
+                      ? <FormattedMessage id='sfoxexchangedata.create.createaccount.partner.verifiedemail' defaultMessage='Verified Email Address' />
```

Comment:
How can you get to this step if your email is not verified?

Reply:
With unverified email, -->  buy/sell tab -->  modal opens to verifyEmail --> click cancel. 
On cancel, we can either close the modal or return user to AcceptTerms. I figured keeping them in the flow is better than closing modal (which already has a close mechanism anyway).

Reply:
Shouldn't it take you back to change email step?